### ISSUE TITLE
Menu items are announced as menus

### ISSUE THREAD

**Issue by danhorvath** [FreedomScientific/standards-support#826]
### Summary

JAWS announces menu items as menus.

1. Go to https://65fa8a6751742e21b1bc65bd-gjdcjkughl.chromatic.com/?path=/story/components-dropdownmenu--dropdown-menu-story
1. Focus the menu button
1. Press ArrowDown

The same issue is there for https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/examples/menu-button-actions/

### Expected result

JAWS announces "Account settings 1 of 8"

### Actual result

JAWS announces "Account settings menu 1 of 8"

### Example
* https://65fa8a6751742e21b1bc65bd-gjdcjkughl.chromatic.com/?path=/story/components-dropdownmenu--dropdown-menu-story
* https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/examples/menu-button-actions/

### Additional Information
It works fine in JAWS 2023 and NVDA 2024.1

#### JAWS version and build number
Version 2024.2403.3

 
#### Operating System and version
Windows 11

#### Browser and version:
Chrome 124

**Comment by JAWS-test** [comment]
I cannot understand the error description. When I activate the button with JAWS 2023 and JAWS 2024, a menu opens and the focus is set to the menu. This is why JAWS first outputs the menu (with its labelling) and then the menu entry. If I navigate through the menu entries with the arrow keys, only the menu entry is output. The output appears to me to be correct

**Comment by danhorvath** [comment]
I do the following:
1. focus the "Setting" button
2. press arrow down to open the menu and focus the "Account settings" item
3. press arrow down again to focus the "Show third party content" item

For this I get the following transcription:
>Settings
button menu collapsed
Settings button menu
Press space to activate the menu, then navigate with arrow keys
Settings menu
Account settings menu
1 of 8
To move through items press up or down arrow.
Show third party content menu checked
3 of 8

Notice how it announces "Account settings menu" instead of "Account settings" and "Show third party content menu checked" instead of "Show third party content checked". I don't seem to remember having the same issue on JAWS2023 (although it's been some time since I last tried it) and NVDA doesn't announce menu items as menus either.

**Comment by JAWS-test** [comment]
I do not use English as the output language of JAWS. This may explain the differences
